### PR TITLE
[Doc] add scneencasts for StackedFiltersForm and update code sample

### PR DESCRIPTION
**Pull request by adguernier**
## Problem

The StackedFiltersForm includes a screenshot but it was missing.

## Solution

Add a screencast to demonstrate StackedFiltersForm and update code sample based on this screencast

### CODE REVIEW

File: docs/StackedFilters.md [hunk 1]
```diff
@@ -346,37 +351,39 @@ import {
 } from '@mui/material';
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
 
-const postListFilters: FiltersConfig = {
+const PostListFilters: FiltersConfig = {
```
Comment (slax57) [inline docs/StackedFilters.md:354]: You should keep the `PostListFilters` starting with a lowercase, as it is a configuration object and not a React Component

```suggestion
const postListFilters: FiltersConfig = {
```

**Review by slax57 — CHANGES_REQUESTED** [review]
Other than that, LGTM!

You will probably need to port this change back to the EE doc as well :grin:

**Review by slax57 — APPROVED** [review]

File: docs/StackedFilters.md [hunk 2]
```diff
@@ -347,36 +352,38 @@ import {
...
+                            {Object.keys(filterValues).length} filter(s) applied
+                        </>
+                    ) : (
+                        'Filters'
+                    )}
+                </Typography>
             </AccordionSummary>
             <AccordionDetails id="filters-content">
-                <StackedFiltersForm config={postListFilters} />
+                <StackedFiltersForm config={PostListFilters} />
```
Comment (fzaninotto) [inline docs/StackedFilters.md:382]: no, this should be a lowercase

File: docs/StackedFilters.md [hunk 3]
```diff
@@ -347,36 +352,38 @@ import {
...
                 expandIcon={<ExpandMoreIcon />}
                 aria-controls="filters-content"
                 id="filters-header"
             >
-                <Typography>Filters</Typography>
+                <Typography>
+                    {Object.keys(filterValues).length ? (
+                        <>
+                            {Object.keys(filterValues).length} filter(s) applied
+                        </>
```
Comment (fzaninotto) [inline docs/StackedFilters.md:375]: use a template instead to make it fit in one line

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: docs/StackedFilters.md (hunk 3 above)
Comment (adguernier) [reply docs/StackedFilters.md]: This snippet comes from Storybook, I did a naive copy/paste 😅

**Review by fzaninotto — APPROVED** [review]

**Pull request merged**